ts/parts/Utilities.ts

```diff
@@ -157,9 +157,23 @@ declare global {
         function arrayMin(data: Array<any>): number;
         function attr(
             elem: (HTMLDOMElement|SVGDOMElement),
-            prop?: (string|HTMLAttributes|SVGAttributes),
+            prop: (HTMLAttributes|SVGAttributes)
+        ): undefined;
+        function attr(
+            elem: (HTMLDOMElement|SVGDOMElement),
+            prop: string,
+            value?: undefined
+        ): (string|null);
+        function attr(
+            elem: (HTMLDOMElement|SVGDOMElement),
+            prop: string,
+            value: (number|string)
+        ): undefined;
+        function attr(
+            elem: (HTMLDOMElement|SVGDOMElement),
+            prop: (string|HTMLAttributes|SVGAttributes),
             value?: (number|string)
-        ): any;
+        ): (string|null|undefined);
```

Comment:
I am not sure, if this one is necessary. It looks like a merge of the three previous declarations.

Reply:
I believe it is only for the actual implementation down in the code needed.

Reply:
Yes, I believe you are right. This function type is not needed, since the more specific overloads will catch it first.
Will make another update.

Reply:
@bre1470 Please see latest changes.